package.json

```diff
@@ -29,6 +29,9 @@
   },
   "homepage": "https://github.com/mixmaxhq/eslint-config-mixmax#readme",
   "peerDependencies": {
-    "eslint": ">= 3"
-  }
+    "babel-eslint": "^7.2.3",
+    "eslint": ">= 3",
+    "eslint-plugin-react": "^7.1.0"
```

Comment:
Also please update the README [here](https://github.com/mixmaxhq/eslint-config-mixmax#usage) with respect to browser configs, it's out of date.